trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/MainDispatcher.java

```diff
@@ -535,7 +535,7 @@ public Result dispatch(final ACSLNode n, final IASTNode cHook) {
 				return mAcslHandler.visit(this, (IfThenElseExpression) n);
 			}
 			if (n instanceof QuantifierExpression) {
-				return mAcslHandler.visit(this, n);
+				return mAcslHandler.visit(this, (QuantifierExpression) n);

```

Comment:
```suggestion
			if (n instanceof QuantifierExpression quantified) {
				return mAcslHandler.visit(this, quantified);
```

Reply:
Yeah, I was tempted to suggest rewriting the entire method (or multiple methods) with pattern-matching switch-cases, but better to keep this PR simple :grin: